lib/data-generator.js

```diff
@@ -19,7 +20,25 @@ module.exports = function(runner, done) {
   });
 
   runner.on('test end', function(test) {
-    var error;
+    let error,
+        result = test.ctx.result;
+
+    // go to the main directory to get the assets
+    if (result) {
+      let baseline = result.baseline,
+          screenshot = result.screenshot,
+          diff = result.diff;
+
+      if (baseline) {
+        result.baseline = path.join('..', '..', baseline);
```

Comment:
Mugshot will create its `rootDirectory` in the directory where you run `mocha test`. The reporter will create the `visual-report` in the same folder and will require the images (that Mugshot created) in `statics/build/bundle.js`.